SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
+     * @param   fileOut <String> storing the file name of the output (SFSP)
+     * @param   strategy <String> storing the method of sampling the random dataset
+     * @param   datasetFilePaths <String[]> storing the file paths of the dataset files
+     * @param   numSfsp <Integer> storing the number of statistically significant sequential patterns
+     * @param   numFsp  <Integer> storing the number of frequent sequential patterns
+     * @param   correctedThreshold <Double> storing the correctedThreshold
+     * @param   positionOrigin <Int2ObjectOpenHashMap<Utils.Pair>> storing numbered positions originally
+     * @param   datasetOrigin <Utils.Dataset> storing the original dataset
```

Comment:
We want to rename any reference to "original" to "observed", both in comments and in variable names.